tests/basic_env.py

```diff
@@ -21,7 +21,55 @@
 logger = logging.getLogger("dvc")
 
 
-class TestDirFixture(object):
+class EmptyDirFixture(object):
```

Comment:
@pared, not sure if it is the same, but what about?

```python
@pytest.fixture
def chdir_tmp(tmp_path):
    old_directory = os.cwd()
    new_directory = fspath_py35(tmp_path)

    os.chdir(new_directory)
    yield
    os.chdir(old_directory)
```

Also, why refactoring something that we will deprecate eventually) ?